extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -11,13 +13,266 @@
+  * Renamed `constant` to `offset`, and `divisor` to `repeat`
+  * Removed `statistics` specification, to be considered as a future extension
+  * Renamed `featureTable` to `propertyTable` and `featureTexture` to `propertyTexture`
+  * Removed `featureIdAttributes` and `featureIdTextures`, replaced with `featureIds`
+    * Primitives and Nodes may now have feature IDs without associated property tables
+  * Removed string ID references to property tables and textures, replaced with integer IDs
+  * Removed `optional` and added `required`. Properties are now assumed to be optional unless `required` is true.
+  * Added `noData` for specifying a sentinel value that indicates missing data
+  * Removed `default`
+  * Feature ID values outside the range `[0, count - 1]` now indicate "no associated feature"
+  * `NaN` and `Infinity` are now explicitly disallowed as property values
+  * Binary alignment, offset, and padding changes:
+    * Byte offsets for buffer views in a GLB-stored BIN chunk are no longer different from the core glTF specification
+    * Relaxed buffer view alignment to component size, rather than strict 8-byte boundaries
+  * Renamed `_FEATURE_ID_#` to `FEATURE_ID_#`
+  * Added vector and matrix types: `VEC2`, `VEC3`, `VEC4`, `MAT2`, `MAT3`, `MAT4`
+  * Refactored `type` and `componentType` to avoid overlap. Properties that store a single value now have a `type` of `SINGLE` and a `componentType` of the desired type (e.g. `type: "SINGLE", componentType: "UINT8"`)
+  * Class IDs, enum IDs, and property IDs must now contain only alphanumeric and underscore characters
+  * Clarified that nodes with GPU instancing cannot reference property textures
+  * For GPU instance metadata, the `EXT_mesh_features` extension is now scoped to the `node` extensions instead of nesting inside the `EXT_mesh_gpu_instancing` extension.
+  * Split `offsetType` into `arrayOffsetType` and `stringOffsetType`
+  * Refactored the property texture schema so it is now a glTF `textureInfo` object.
+  * Each property texture now contains only a single texture
+  * Property textures are now assumed to be in linear space, and must use nearest or linear filtering
+  * Added a `schema.id` property
+* **Version 3.0.0** February 2022
+  * Elements in the property texture `properties` dictionary are now objects containing a `channels` property rather than an array of channels directly.
+  * Split the `EXT_mesh_features` extension into one that only defines the concept of feature IDs, and a separate `EXT_structural_metadata` extension that allows associating the features with metadata, including the following changes:
+    * Renamed `FEATURE_ID_#` back to `_FEATURE_ID_#`
+    * Removed the "Implicit Feature IDs" concept
+    * JSON restructuring (inlined `propertyTable`, store `texture` and `attribute` as mutually exclusive properties (instead of different types), added `featureCount` and `nullFeatureId`, allow multiple texture channels to be combined into one feature ID)
```

Comment:
Other things to add:

* Removed instanced feature IDs (that will become `EXT_instance_features`)
* If we decide to make `featureIds` a dictionary that should be included here

Reply:
Last commit links to the "common" change log